markdown-link-check.json

```diff
@@ -46,10 +46,10 @@
       "pattern": "^https://analytics.amplitude.com"
     },
     {
-      "pattern": "^https://stackoverflow.com/questions/tagged/ddev"
+      "pattern": "^https://(meta.stackoverflow|stackoverflow|superuser).com/"
```

Comment:
```suggestion
      "pattern": "^https://(meta.)?stackoverflow|superuser).com/"
```

Reply:
Thanks, more elegant and much appreciated, but I think the less elegant one is easier to understand and expand when the time comes.